cmc-domain-sample-data/build.gradle

```diff
@@ -0,0 +1,16 @@
+group 'uk.gov.hmcts.cmc'
```

Comment:
I don't think there's a need to be that granular, sample data is perfectly fine to live along the domain. It's even better because it's easier to see you need to update sample data builders when changing the model.

Reply:
It was previously part of `test/java` not the `main/java`. The reason I pulled out is, it is required in domain-model tests and other tests (services & controllers in parent module) and we can add testCompile dependency. I prefer not to use name Sample***.java for Builder classes. If these are builder classes only, these should have proper names. Probably another cleanup on these.

Reply:
they aren't builder classes, they are sample's because they provide defaults for tests